packages/cbioportal-frontend-commons/src/components/gradientLegend/GradientLegend.tsx

```diff
@@ -61,6 +61,7 @@ export class GradientLegend extends React.Component<IGradientLegendProps> {
 
         return [
             <text
+                key="gradientLegendLabelMax"
```

Comment:
is this something that coule appear more than once on a page at the same time?

Reply:
arent keys only relevant within a single list, for react?

Reply:
Just double checked. I see this error message in the console if I remove the key: 
```Warning: Each child in a list should have a unique "key" prop.```

I think it is because we are returning an array here.